### ISSUE TITLE
Invalid specification for coordinate_range values

### ISSUE THREAD

**Issue by kenshaw** [TinyVG/specification#23]
The `coordinate_range` field is specified as `Default` (8-bit) and `Reduced` (16-bit), however, the reference implementation is flipped, using `u8` for `reduced` and `u16` for `default` -- from `lib/parsing.zig`:

```zig
                      const width: u32 = switch (range) {    
                          .reduced => mapZeroToMax(try reader.readIntLittle(u8)),    
                          .default => mapZeroToMax(try reader.readIntLittle(u16)),    
                          .enhanced => std.math.cast(u32, mapZeroToMax(try reader.readIntLittle(u32))) catch return error.InvalidData,    
                      };    
                      const height: u32 = switch (range) {    
                          .reduced => mapZeroToMax(try reader.readIntLittle(u8)),    
                          .default => mapZeroToMax(try reader.readIntLittle(u16)),    
                          .enhanced => std.math.cast(u32, mapZeroToMax(try reader.readIntLittle(u32))) catch return error.InvalidData,    
                      };  
```

Screenshot from `specification.pdf`:
![Screenshot from 2021-12-23 11-09-21](https://user-images.githubusercontent.com/2274859/147186909-8b2011a1-7515-45c7-aab1-78d29c8150fe.png)

**Comment by kenshaw** [comment]
This appears to have been fixed in 49ba3c0b. I was working off an old PDF.

**Comment by ikskuh** [comment]
> This appears to have been fixed in 49ba3c0. I was working off an old PDF.

Yeah, you're not the first person finding this. Thanks for the report!
